### ISSUE TITLE
Demo not working on windows 10

### ISSUE THREAD

**Issue by Moneyl** [ebruneton/precomputed_atmospheric_scattering#37]
Hi. I've tested the web demo on firefox, chrome, and edge, and built the demo locally as well. Both of them show the red text describing the controls in the top left corner with the rest of the screen being black. I've tried waiting several minutes and playing with the controls and still it just shows the controls and a black screen.

I'm on windows 10 version 1909. Using a gtx 1070 so I don't believe it's a hardware incompatibility problem. When I compiled it locally I used visual studio 2019.

**Comment by Moneyl** [comment]
I just noticed I'm getting some shader compile errors when running the demo locally. Will see if I can fix them:
```
compile log = 0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(36) : error C1503: undefined variable "ComputeTransmittanceToTopAtmosphereBoundaryTexture"
0(37) : error C1503: undefined variable "ATMOSPHERE"

link log = Fragment info
-------------
0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(36) : error C1503: undefined variable "ComputeTransmittanceToTopAtmosphereBoundaryTexture"
0(37) : error C1503: undefined variable "ATMOSPHERE"
(0) : error C2003: incompatible options for link

compile log = 0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(38) : error C1503: undefined variable "ComputeDirectIrradianceTexture"
0(39) : error C1503: undefined variable "ATMOSPHERE"

link log = Fragment info
-------------
0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(38) : error C1503: undefined variable "ComputeDirectIrradianceTexture"
0(39) : error C1503: undefined variable "ATMOSPHERE"
(0) : error C2003: incompatible options for link

compile log = 0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(42) : error C1503: undefined variable "ComputeSingleScatteringTexture"
0(43) : error C1503: undefined variable "ATMOSPHERE"

link log = Fragment info
-------------
0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(42) : error C1503: undefined variable "ComputeSingleScatteringTexture"
0(43) : error C1503: undefined variable "ATMOSPHERE"
(0) : error C2003: incompatible options for link

compile log = 0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(43) : error C1503: undefined variable "ComputeScatteringDensityTexture"
0(44) : error C1503: undefined variable "ATMOSPHERE"

link log = Fragment info
-------------
0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(43) : error C1503: undefined variable "ComputeScatteringDensityTexture"
0(44) : error C1503: undefined variable "ATMOSPHERE"
(0) : error C2003: incompatible options for link

compile log = 0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(42) : error C1503: undefined variable "ComputeIndirectIrradianceTexture"
0(43) : error C1503: undefined variable "ATMOSPHERE"

link log = Fragment info
-------------
0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(42) : error C1503: undefined variable "ComputeIndirectIrradianceTexture"
0(43) : error C1503: undefined variable "ATMOSPHERE"
(0) : error C2003: incompatible options for link

compile log = 0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(42) : error C1503: undefined variable "ComputeMultipleScatteringTexture"
0(43) : error C1503: undefined variable "ATMOSPHERE"
0(47) : error C1503: undefined variable "RayleighPhaseFunction"

link log = Fragment info
-------------
0(16) : error C0000: syntax error, unexpected identifier, expecting "::" at token "AtmosphereParameters"
0(42) : error C1503: undefined variable "ComputeMultipleScatteringTexture"
0(43) : error C1503: undefined variable "ATMOSPHERE"
0(47) : error C1503: undefined variable "RayleighPhaseFunction"
(0) : error C2003: incompatible options for link
```

**Comment by Moneyl** [comment]
I don't know why the web demo isn't working for me, but I got my local build to work. It seems the issue was due to the path I cloned the repo to containing spaces. I had it in a folder called "_Downloaded repos". When I ran the build scripts I got an error `'C:\Users\moneyl\source\repos\_Downloaded' is not recognized as an internal or external command,
operable program or batch file`. Since it was ending the path at the space I figured the space in the path might be causing shader loading issues as well. Re-cloning the repo to a path without spaces fixed the issue. 

As long as the web demo is working for others I wouldn't worry about that. My internet is slow and unreliable so that could be the reason for the web demo not working. The demo looks great, thanks for making this. Closing this issue.
